code/src/java/pcgen/persistence/lst/CampaignLoader.java

```diff
@@ -176,6 +176,9 @@ private void initRecursivePccFiles(Campaign baseCampaign,
         baseCampaign.addAllToListFor(ListKey.FILE_EQUIP_MOD, subCampaign.getSafeListFor(ListKey.FILE_EQUIP_MOD));
         baseCampaign.addAllToListFor(ListKey.FILE_KIT, subCampaign.getSafeListFor(ListKey.FILE_KIT));
         baseCampaign.addAllToListFor(ListKey.FILE_BIO_SET, subCampaign.getSafeListFor(ListKey.FILE_BIO_SET));
+        baseCampaign.addAllToListFor(ListKey.FILE_ALIGNMENT, subCampaign.getSafeListFor(ListKey.FILE_ALIGNMENT));
```

Comment:
Could this be done using a loop over a set of ListyKeys instead? That would make it a bit safer and easier to expand in the future